sdk/loadtestservice/load-testing-rest/samples/v1-beta/javascript/README.md

```diff
@@ -0,0 +1,63 @@
+---
+page_type: sample
+languages:
+  - javascript
+products:
+  - azure
+  - azure-load-testing
+urlFragment: load-testing-javascript-beta
+---
+
+# Azure Load Testing rest client library samples for JavaScript (Beta)
+
```

Comment:
Can we add some intro like this?
Azure Load Testing provides client library in JavaScript to the user by which they can interact natively with Azure Load Testing service. Azure Load Testing is a fully managed load-testing service that enables you to generate high-scale load. The service simulates traffic for your applications, regardless of where they're hosted. Developers, testers, and quality assurance (QA) engineers can use it to optimize application performance, scalability, or capacity.